transformation/src/transformationEndpoint.ts

```diff
@@ -16,10 +20,12 @@ export class TransformationEndpoint {
   keycloak?: Keycloak
 
   transformationService: TransformationService
+  
 
   constructor (transformationService: TransformationService, port: number, auth: boolean) {
```

Comment:
Pass storageHandler and amqpHandler as constructor parameters and initialize outside. Let's us insert mocks for testing